### DISCUSSION TITLE
文档想部署的时候需要登录功能才能进入文档应当如何做?

### DISCUSSION THREAD

**Discussion by Silence-dream** [umijs/dumi#1574] [Q&A] [answered]
有什么解决方案吗?

**Comment by PeachScript** [comment, marked as answer]
`.dumi/app.ts` 里可以写运行时钩子，在钩子里做权限判定决定执行 render 还是跳转到登录系统，参考：https://umijs.org/docs/api/runtime-config#renderoldrender-function
